.env.template

```diff
@@ -25,6 +25,7 @@ inxtbridge_stripe__SK_TEST=sk_test_stripe_key
 inxtbridge_complex__rpcUrl=http://landlord:8080
 inxtbridge_mailer__auth__user=internxt
 inxtbridge_mailer__sendgrid__api_key=APIKEY
+inxtbridge_mailer__sendgrid__delete_template_id=d-d949263926b64774a2741307db2db431
```

Comment:
also, no problem to use this publicly?

Reply:
It's ok if they don't have the Sendgrid api key, but I'd better remove it